supabase/migrations/20240409135335_add_website_data_and_status_orders.sql

```diff
@@ -0,0 +1,31 @@
+DELETE FROM public.website_data;
```

Comment:
We're happy to just delete all items from website_data and status_order since we know the exact values we need them to be for now.

In the future
- Adding to website_data should be done using insert like in any other table
- Adding to status_order is a bit more tricky, as we'd likely want to reshuffle the order too. We'd probably end up deleting all and defining all as that's the easiest thing to do. (We could instead define a logic to add one item at a certain position and shift the order number up by one for some existing items, but might be an overkill.)